src/rdb.h

```diff
@@ -140,6 +140,9 @@
 #define RDB_LOAD_ERR_EMPTY_KEY 1 /* Error of empty key */
 #define RDB_LOAD_ERR_OTHER 2     /* Any other errors */
 
+/* Size of the static buffer used for rdbcompression */
+#define LZF_STATIC_BUFFER_SIZE (8 * 1024)
```

Comment:
consider moving `LZF_STATIC_BUFFER_SIZE` to `rdb.c` since it's only used internally